compiler/qsc/src/interpret/stateful/tests.rs

```diff
@@ -48,7 +50,16 @@ mod given_interpreter {
                 .expect("interpreter should be created");
 
                 let (result, output) = line(&mut interpreter, "Message(\"_\")");
-                is_only_error(&result, &output, "name error: `Message` not found");
+                is_only_error(
```

Comment:
I added some additional validation to the existing test cases to make sure source spans were getting formatted correctly.